gui/middleware/notifier.py

```diff
@@ -5982,6 +5986,10 @@ def fc_enabled(self):
         return fc_enabled()
 
 
+def crypt_makeSalt():
+    return '$6$' + str(uuid.uuid4().get_hex()[0:16])
+
```

Comment:
Using uuid4 does not sound like a good idea, salt will accept many more characters, uppercase a-z, slash and dot.
I think you should use a randomly chosen values of a string with all possible values.